src/components/pages/MarkdownEditor.jsx

```diff
@@ -81,7 +92,20 @@ const MarkdownEditor = ({
   const events = useMemo(() => {
     return {
       paste: (cm, e) => {
-        const convertedText = toMarkdown(e.clipboardData.getData("text/html"))
+        const pasteText = e.clipboardData.getData("text/plain")
+        const processedPasteText = processInstagramEmbedToTag(pasteText)
+
+        // If the paste text contains an Instagram embed, replace the paste text
+        // with the processed paste text.
+        // Note: This will prevent further processing of the paste text, as
+        // custom HTML tags will be removed by Turndown
+        if (processedPasteText !== pasteText) {
+          e.preventDefault()
+          cm.replaceSelection(processedPasteText)
+          return
+        }
+
+        const convertedText = toMarkdown(pasteText)
```

Comment:
Hmm I tried the original way of using `text/html` but nothing seems to be returned from this `toMarkdown` function. I switched to `text/plain` after some trial-and-error.

Reply:
could you elaborate on this? from examining the function inside `useMarkdown.ts`, `toMarkdown` converts html into markdown - a litmus test for this would be to open up your local FE and paste the first example
 from [here](https://www.w3schools.com/html/html_tables.asp). this should result in a similar output as follows: <img width="503" alt="Screenshot 2022-09-06 at 10 44 41 AM" src="https://user-images.githubusercontent.com/44049504/188536133-20509203-dc96-4e02-a90b-3e49031a815b.png">

if the code here passes this test, please resolve this!

Reply:
We're not proceeding with this PR but wanted to comment on this issue. I tried out pasting the table and seems to only work for `text/plain`. Here's a video demo:

https://user-images.githubusercontent.com/27919917/189013172-e90c2b45-1e3d-4fca-92a8-650da97a6df2.mp4

I'm not sure if this bug only happens on my end?

Reply:
hmm you might've copy pasted the wrong one HAHAHA try copy pasting the example
![Screenshot 2022-09-08 at 1 19 16 PM](https://user-images.githubusercontent.com/44049504/189040377-8d545ae6-7c53-4f15-84e0-c6e422225385.png)

Reply:
Ohh I think I know why, I think if using `text/html`, we can just copy the output directly and paste into the editor, which will automatically convert to Markdown. For `text/plain`, someone pasting in the raw HTML code will have it automatically converted to Markdown.

I think let's retain the current behaviour? And thanks for the clarification!